disciplines/isq.ttl

```diff
@@ -626,6 +1401,83 @@ Entropy"""@en ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "9-12.2" ;
+                                           skos:altLabel "StandardConcentration"@en ,
+                                                         "StandardMolarConcentration"@en ;
+                                           skos:prefLabel "StandardAmountConcentration"@en .
+
+
+###  http://emmo.info/emmo#EMMO_476cb776_8219_418d_92e8_2fe04b1fe5cf
+:EMMO_476cb776_8219_418d_92e8_2fe04b1fe5cf rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_dae32a4a_d8da_4047_81b0_36a9713fdce1 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_3227b821_26a5_4c7c_9c01_5c24483e0bd0
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "5-25.2" ;
+                                           skos:prefLabel "MaximumEfficiency"@en .
+
+
+###  http://emmo.info/emmo#EMMO_477042ef_3173_49b2_a264_0e3270733f8c
+:EMMO_477042ef_3173_49b2_a264_0e3270733f8c rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_b3600e73_3e05_479d_9714_c041c3acf5cc
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-38.2" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "distance in a superconductor over which the effect of a perturbation is appreciable at zero thermodynamic temperature"@en ;
+                                           skos:prefLabel "CoherenceLength"@en .
+
+
+###  http://emmo.info/emmo#EMMO_47f2da35_3b01_4290_964a_589c593e85c6
+:EMMO_47f2da35_3b01_4290_964a_589c593e85c6 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_dae32a4a_d8da_4047_81b0_36a9713fdce1 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_3ecff38b_b3cf_4a78_b49f_8580abf8715b
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "5-23" ;
```

Comment:
Do you have an elucidation for PlanckFunction?

Reply:
done